src/View/StringTemplate.php

```diff
@@ -339,13 +340,16 @@ protected function _formatAttribute(string $key, $value, $escape = true): string
     /**
      * Adds a class and returns a unique list either in array or space separated
      *
-     * @param array|string $input The array or string to add the class to
-     * @param array|string $newClass the new class or classes to add
+     * @param \stdClass|array|string|bool|null $input The array or string to add the class to
+     * @param array|string|bool|null $newClass the new class or classes to add
      * @param string $useIndex if you are inputting an array with an element other than default of 'class'.
-     * @return array<string>|string
+     * @return array<string>|string|null
      */
-    public function addClass($input, $newClass, string $useIndex = 'class')
-    {
+    public function addClass(
+        stdClass|array|string|bool|null $input,
+        array|string|bool|null $newClass,
+        string $useIndex = 'class'
+    ): array|string|null {
```

Comment:
I just added all the types that were being passed in. It requires the suppression below because $newClass might not be array ... due to bool, I think. Not sure if if `empty($newClass)` is ever false.

Reply:
Couldn't it be false if an empty string or `null` is passed?

Reply:
I'm just saying, the original cod doesn't detect or handle a false where I added the suppression. Not sure if it's ever encountered.